app/assets/javascripts/modules/auto_logoff.js.coffee

```diff
@@ -14,11 +14,17 @@ class app.AutoLogoff
     setInterval(( -> logoff_timer()), 1000)
 
   logoff_timer = () ->
-    if document.URL.indexOf('/login/login') > -1
+    if document.URL.indexOf('/sessions/new') > -1
       return
     if remaining_seconds <= 1
-      window.location = '/login/logout?jumpto=' + window.location.pathname
-      Turbolinks.pagesCached(0)
+      $.ajax({
+        method: "DELETE",
+        url: '/sessions',
+        data: {
+          message: I18n.t('sessions.destroy.expired'),
```

Comment:
use autologout: true instead of translated message.
if autologout: true present, display translated message. I think it's not a good idea to send translated text like this to the backend and then evaluate it there. the autologout makes it more bullet proof.